Shared/Features/Freaks/View/FilterView.swift

```diff
@@ -1,23 +1,29 @@
 import SwiftUI
 
-struct SkillsView: View {
-    @ObservedObject var viewModel: FreaksViewModel
+struct FilterView: View {
+    @ObservedObject var viewModel: FilterViewModel
     @Binding var showSkillsView: Bool
+    @Binding var showProjectsView: Bool
```

Comment:
We can use just one boolean to track the view presented/closed state to simplify the view logic.